[No QA] Update help docs for Release 2: the new free Submit workspace (#92944)
Expensify/App: docs/articles/new-expensify/getting-started/Submit-Workspace.md

---
title: Submit Workspace

> description: Learn how to use a free Submit workspace in New Expensify to code expenses, submit them to an approver by email, and track which reports have been paid.

Reviewer: @MelvinBot 
> by email

Unnecessary, remove it

keywords: [New Expensify, Submit workspace, free plan, get paid back, submit expenses, approver, Editor role, mark as paid]
internalScope: Audience is employees who want to get reimbursed by their employer. Covers creating a Submit workspace, submitting expenses to an approver, and tracking payment. Does not cover Collect or Control workspace configuration.
---

> A Submit workspace is a free workspace in New Expensify for employees who want to get reimbursed by their employer. You can scan receipts, code your expenses with categories and tags, and submit reports to an approver by email — even if your company hasn't adopted Expensify yet. There's no cost to submit, and your approver doesn't need an account to receive your report.

Reviewer: @MelvinBot 
> by email 

Unnecessary, remove.

A Submit workspace includes a limited set of features focused on submitting expenses. Premium features like approvals, payments, company cards, and accounting integrations require upgrading to a paid plan.

---

## Who can use a Submit workspace

> - Anyone can create a free Submit workspace. You become the workspace owner and an Editor.

Reviewer: >  You become the workspace owner and an Editor.

This isn't relevant to the user. Remove it.

- Anyone on your company's email domain can join your Submit workspace and is added as an **Editor**.

> - An **Editor** can configure the workspace (such as adding categories and tags) but cannot see other members' expenses or reports, and cannot change member roles or invite admins.

Reviewer: > or invite admins.

We don't want to emphasize the concept of admins or owners here. Remove "or invite admins"

Premium features are not included in a Submit workspace. To use approvals, payments, company cards, the Expensify Card, Per Diem, Expensify Travel, invoices, or accounting integrations, you'll need to [learn how to upgrade your workspace plan](/articles/new-expensify/billing-and-subscriptions/Changing-Your-Workspace-Plan).

---

## How to create a Submit workspace

You can create a Submit workspace during onboarding or from the pricing page.

During onboarding:

1. Sign up for New Expensify.

> 2. When asked what you want to do, select **Get paid back by my employer**.

Reviewer: > Get paid back by my employer

This has been updated to "Submit expenses to my employer"

3. If no one on your email domain already has a workspace you can join, a Submit workspace is created for you automatically.

From the pricing page:

1. Go to the Expensify pricing page.
2. Find the Submit plan.
3. Click **Get Started**.

After your workspace is created, you're taken to your workspace **Categories** with the **#admins** room open so you can start configuring the workspace or submit your first expense.

> <!-- SCREENSHOT:

Reviewer: Unnecessary, doesn't really show anything helpful.

Suggestion: New Submit workspace showing the Categories page with the #admins room open
Location: After the create-a-Submit-workspace steps
Purpose: Helps the user recognize the post-creation state
-->

---

## How to join an existing Submit workspace

If someone on your email domain already owns a Submit workspace, you can join it when you sign up:

1. Sign up for New Expensify.
2. Enter your name and validate your login.
3. Select the Submit workspace from the list of available workspaces to join it.

When you join, you're added as an **Editor**. You can collaborate on the workspace configuration, but you can't see or take action on other members' expenses or reports.

---

## How to submit an expense to an approver

When your report is ready, you submit it to the person who approves your expenses by entering their email:

1. Add your expenses to a report.
2. Click **Submit**.
3. Select an existing workspace member, or enter the email address of your approver.
4. Confirm to submit the report.

> Your approver is remembered for next time, but you can change it before any submission. Your approver receives an email with an **Approve** button and a PDF copy of the report. They don't need an Expensify account to receive it.

Reviewer: > Your approver is remembered for next time, but you can change it before any submission.

This sounds awkward. I think we can remove it and set this as point 5:
> Your approver receives an email asking them to approve your expenses and a PDF copy of the report.

---

## How to mark a report as paid

If you've been reimbursed outside of Expensify, you can mark an outstanding report as paid so you can track its status yourself:

1. Open the outstanding report.
2. Select **Mark as paid**.

Marking a report as paid lets you track which of your submitted reports are still outstanding without requiring your approver to sign in. After a report is marked as paid, your approver can no longer take action on it.

---

## What happens when your approver approves a report

When your approver clicks **Approve**, they're prompted to upgrade the workspace to a paid plan to complete the approval. If they upgrade:

- The workspace upgrades to the Collect plan and the approver becomes the workspace owner and admin.
- All other members, including you, are set to the **Member** role.
- You'll receive a message from Concierge letting you know the workspace was upgraded.

> Until an approver upgrades, you can keep using the Submit workspace for free.

Reviewer: I think this last point is unnecessary, let's remove it.

---

A Submit workspace is a free workspace in New Expensify for employees who want to get reimbursed by their employer. You can scan receipts, code your expenses with categories and tags, and submit reports to an approver — even if your company hasn't adopted Expensify yet. There's no cost to submit, and your approver doesn't need an account to receive your report.

Premium features are not included in a Submit workspace. To use approvals, payments, company cards, the Expensify Card, Per Diem, Expensify Travel, invoices, or accounting integrations, you'll need to [learn how to upgrade your workspace plan](https://github.com/articles/new-expensify/billing-and-subscriptions/Changing-Your-Workspace-Plan).

---

## Who can use a Submit workspace

- Anyone can create a free Submit workspace.
- Anyone on your company's email domain can join your Submit workspace and is added as an **Editor**.
- An **Editor** can configure the workspace (such as adding categories and tags) but cannot see other members' expenses or reports, and cannot change member roles.

---

## How to create a Submit workspace

> You can create a Submit workspace during onboarding or from the pricing page.

Reviewer 2: ```suggestion
You can create a Submit workspace when creating your Expensify account or from the Expensify pricing and plans page.
```

> During onboarding:

Reviewer 2: ```suggestion
When creating your Expensify account:
```

1. Sign up for New Expensify.

> 2. When asked what you want to do, select **Submit expenses to my employer**.

Reviewer 2: ```suggestion
2. When asked **What do you want to do today?**, select **Submit expenses to my employer**.
```

3. If no one on your email domain already has a workspace you can join, a Submit workspace is created for you automatically.

> From the pricing page:

Reviewer 2: ```suggestion
If you already have an Expensify account:
```

> 1. Go to the Expensify pricing page.

Reviewer 2: ```suggestion
1. Go to the [Expensify pricing and plans page](www.expensify.com/pricing).
```

2. Find the Submit plan.

> 3. Click **Get Started**.

Reviewer 2: ```suggestion
3. Click **Get started**.
```

## What happens after you create a Submit workspace

- You can configure workspace settings, like categories and tags.
- You can submit reports for approval.
- Coworkers on the same email domain can join your workspace.

Members who join a Submit workspace are assigned the Editor role.

Editors can:
- Configure shared workspace settings, including categories and tags.
- Create and submit their own reports.
 
Editors can't:
- View other members' expenses or reports.
- Change member roles.

---

## How to join an existing Submit workspace

> If someone on your email domain already owns a Submit workspace, you can join it when you sign up:

Reviewer 2: ```suggestion
If someone on your email domain already owns a Submit workspace, you can join it when creating your Expensify account: 
```

1. Sign up for New Expensify.
2. Enter your name and validate your login.

> 3. Select the Submit workspace from the list of available workspaces to join it.

Reviewer 2: ```suggestion
1.  Create an Expensify account.
2. Complete the prompts to set up your account.
3. On the step **See if your team's in Expensify**, enter the validation code sent to your email address.
4. If there are existing workspaces associated with your email domain, you'll be prompted to join one. Select the Submit workspace.``

When you join, you're added as an **Editor**. You can collaborate on the workspace configuration, but you can't see or take action on other members' expenses or reports.

> 

Reviewer 2: ```suggestion

If you weren't automatically added to the workspace when you created your account, ask a current Editor of the Submit workspace to invite you to join. Learn how to [invite members to a workspace](/articles/new-expensify/workspaces/Managing-Workspace-Members#invite-members-to-a-workspace).
```

---

> title: Using our free Submit Workspace

Reviewer 2: ```suggestion
title: Use a Free Submit Workspace
```

description: Learn how to use a free Submit workspace in New Expensify to code expenses, submit them to an approver, and track which reports have been paid.
keywords: [New Expensify, Submit workspace, free plan, get paid back, submit expenses, approver, Editor role, mark as paid]
internalScope: Audience is employees who want to get reimbursed by their employer. Covers creating a Submit workspace, submitting expenses to an approver, and tracking payment. Does not cover Collect or Control workspace configuration.
---

A Submit workspace is a free workspace in New Expensify for employees who want to get reimbursed by their employer. You can scan receipts, code your expenses with categories and tags, and submit reports to an approver — even if your company hasn't adopted Expensify yet. There's no cost to submit, and your approver doesn't need an account to receive your report.

> 

Reviewer 2: ```suggestion

Premium features are not included in a Submit workspace. To use approvals, payments, company cards, the Expensify Card, Per Diem, Expensify Travel, invoices, or accounting integrations, you'll need to [learn how to upgrade your workspace plan](https://github.com/articles/new-expensify/billing-and-subscriptions/Changing-Your-Workspace-Plan).

```

---

## Who can use a Submit workspace

- Anyone can create a free Submit workspace.
- Anyone on your company's email domain can join your Submit workspace and is added as an **Editor**.
- An **Editor** can configure the workspace (such as adding categories and tags) but cannot see other members' expenses or reports, and cannot change member roles.

> Premium features are not included in a Submit workspace. To use approvals, payments, company cards, the Expensify Card, Per Diem, Expensify Travel, invoices, or accounting integrations, you'll need to [learn how to upgrade your workspace plan](/articles/new-expensify/billing-and-subscriptions/Changing-Your-Workspace-Plan).

Reviewer 2: ```suggestion

```

^^ I think this belongs in the intro instead - suggested it there!

---

> ## How to create a Submit workspace

Reviewer 2: @garrettmknight for someone who already has an Expensify account, is the pricing page the only way to create the Submit workspace? Or can you create one from the Workspaces tab? 

I think it may be a little clearer if we have the steps list out whatever the primary workspace creation flow is, then the other methods can be covered in a note.

Reviewer: Pricing page is the only way outside of the onboarding flow.

Reviewer: The primary flow is in onboarding. The pricing page is more of an escape hatch, so we can recommend creating a Submit WS if they chose the wrong intent.

Reviewer 2: Ok, I think it's fine to keep both. Made some suggestions to align with UI naming and user-facing terms.

You can create a Submit workspace during onboarding or from the pricing page.

During onboarding:

1. Sign up for New Expensify.
2. When asked what you want to do, select **Submit expenses to my employer**.
3. If no one on your email domain already has a workspace you can join, a Submit workspace is created for you automatically.

From the pricing page:

1. Go to the Expensify pricing page.
2. Find the Submit plan.
3. Click **Get Started**.

> After your workspace is created, you're taken to your workspace **Categories** with the **#admins** room open so you can start configuring the workspace or submit your first expense.

Reviewer 2: ```suggestion

## What happens after you create a Submit workspace

- You can configure workspace settings, like categories and tags.
- You can submit reports for approval.
- Coworkers on the same email domain can join your workspace.

Members who join a Submit workspace are assigned the Editor role.

Editors can:
- Configure shared workspace settings, including categories and tags.
- Create and submit their own reports.
 
Editors can't:
- View other members' expenses or reports.
- Change member roles.

```

---

## How to join an existing Submit workspace

If someone on your email domain already owns a Submit workspace, you can join it when you sign up:

1. Sign up for New Expensify.
2. Enter your name and validate your login.
3. Select the Submit workspace from the list of available workspaces to join it.

> When you join, you're added as an **Editor**. You can collaborate on the workspace configuration, but you can't see or take action on other members' expenses or reports.

Reviewer 2: @garrettmknight is this any different than the `People you may know are already here!` list of workspaces? If so, I think this fits better in an FAQ Or is there any other way to join an existing Submit workspace -- like could a current workspace member invite them?

Reviewer: Current WS member can invite or they can join on sign up.

Reviewer 2: Ok! Added a note to the bottom to include that as an alternative path.

---

## How to submit an expense to an approver

> When your report is ready, you submit it to the person who approves your expenses by entering their email:

Reviewer 2: ```suggestion
Submit workspaces let you submit reports directly to the person who approves your expenses by entering their email. Your approver doesn't need an Expensify account to receive the report.
```

> 1. Add your expenses to a report.

Reviewer 2: ```suggestion

If you haven't created your report yet, [learn how to create and submit reports](/articles/new-expensify/reports-and-expenses/Create-and-Submit-Reports). 

To submit expenses to an approver: 

1. Open the report you want to submit.
```

2. Click **Submit**.
3. Select an existing workspace member, or enter the email address of your approver.
4. Confirm to submit the report.

> 5. Your approver receives an email asking them to approve your expenses and a PDF copy of the report. They don't need an Expensify account to receive it.

Reviewer 2: I agree, i think we just move this out of the steps and to the bottom of this section.

---

## How to mark a report as paid

If you've been reimbursed outside of Expensify, you can mark an outstanding report as paid so you can track its status yourself:

1. Open the outstanding report.
2. Select **Mark as paid**.

Marking a report as paid lets you track which of your submitted reports are still outstanding without requiring your approver to sign in. After a report is marked as paid, your approver can no longer take action on it.

---

## What happens when your approver approves a report

When your approver clicks **Approve**, they're prompted to upgrade the workspace to a paid plan to complete the approval. If they upgrade:

- The workspace upgrades to the Collect plan and the approver becomes the workspace owner and admin.
- All other members, including you, are set to the **Member** role.

> - You'll receive a message from Concierge letting you know the workspace was upgraded.

Reviewer 2: Since this article is ws creation/member focused, this feels sort of out of scope for this article. Perhaps we make it an FAQ instead, what do you think @garrettmknight? 

Suggestion: 

## What happens when my approver clicks Approve?

If your approver clicks **Approve**, they're prompted to upgrade the workspace to a paid plan before they can complete the approval.

If they upgrade:

- The workspace is upgraded to the Collect plan.
- The approver becomes the workspace owner and admin.
- All existing members, including you, are assigned the **Member** role.
- You'll receive a message from Concierge letting you know the workspace was upgraded.

If they don't upgrade:  
 - They still receive the report by email and can view the attached PDF, but they can't change the report status to **Approved**.
